modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -83,47 +92,81 @@ dynamicPlugins:
+          priority: 100 # <10>
 ----
+  <1> `default._<menu_group_parent_item_name>_`: Enter the menu group parent item name to configure static main menu items.
+  <2> `icon`: Enter the menu icon. Required for parent menu items.
+  <3> `title`: Enter the menu group title. Required for parent menu items.
+  <4> `priority`: (Optional) Enter the order of this menu item within its menu level.
+  <5> `default._<menu_item_name>_`: Enter the menu item name for which you want to override the default value. Add the `default.` prefix to identify a main menu item.
+  <6> `parent`: (Optional) Enter the parent menu item for this item. Required if <menu_item_name> is specified as the child of any menu items.
+  <7> `icon`: (Optional) Enter the menu icon if you do not want to use the default icon.
+  <8> `title`: (Optional) Enter the menu group title. Only required for adding a new custom main menu item. To hide a default main menu item title from the sidebar, set the title as an (`" "`) empty string.
+// Update <8> for release 1.6 as this option (currently a  workaround) would be added as a functionality. RHIDP-6333.
+  <9> `to`: (Optional) Enter the path that the menu item navigates to. If it is not set, it defaults to the home page.
+  <10> `priority`: (Optional) Enter the order of this menu item within its menu level.
 
++
+.Example `mainItems` configuration
 
-<1> The `default.` prefix identifies an item as a main menu item. You can add the `default.` prefix to both individual menu items or parent menu group configuration, such as `default.<menu_group_parent_item_name>` in the previous example.
-
-[NOTE]
-====
-The default priority of main menu items determines their order in the sidebar. You can customize the order of the static main menu items by adjusting their priority values. Ensure that the priority and title of each item is clear to facilitate easy reordering. 
-====
---
-
-
-
-
-
-
-
+[source,yaml]
+----
+default.main-menu-items:
+      menuItems:
+        default.catalog:
+          icon: category # <1>
+          title: My Catalog 
+          priority: 5
+        default.learning-path: 
+          title: '' # <2> to hide the learning path from default sidebar
+        default.parentlist: # <3>
+          title: Overview 
+          icon: bookmarks
+        default.home:
+          parent: default.parentlist # <4>
+        default.references:
+          title: References # <5>
+          icon: school # <6>
+          to: /references # <7>
+----
+  <1> `icon`: Specify if you want to change the icon default menu item for the catalog.
+  <2> `title`: Specify an empty string `" "` to hide the learning path from the default sidebar.
+  <3> `default.parentlist`: Introduces the parent menu item.
+  <4> `parent`: Nests home menu under the `default.parentlist` parent menu item.
+  <5> `title`: Specifies a name for `default.references`
+  <6> `icon`: Displays the `school` icon.
+  <7> `to`: Redirects `default.references` to the `/references` page.
```

Comment:
Asciidoc syntax mistake. The two leading empty spaces denote a literal block, which is not what you intend. See https://docs.asciidoctor.org/asciidoc/latest/verbatim/literal-blocks/#indent-method

```suggestion
<1> `icon`: Specify if you want to change the icon default menu item for the catalog.
<2> `title`: Specify an empty string `" "` to hide the learning path from the default sidebar.
<3> `default.parentlist`: Introduces the parent menu item.
<4> `parent`: Nests home menu under the `default.parentlist` parent menu item.
<5> `title`: Specifies a name for `default.references`
<6> `icon`: Displays the `school` icon.
<7> `to`: Redirects `default.references` to the `/references` page.
```